portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,57 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+document.addEventListener("DOMContentLoaded", function() { 
+  let currSlide = 1;
+  let numImgs = 3;
+
+  createSlide(currSlide, numImgs);
+
+  const prev = document.getElementById("prev");
+  const next = document.getElementById("next");
+
+  function changeSlide (direction) {
+    currSlide += direction;
+    if(currSlide < 1) {
+      currSlide = numImgs;
+    }
+    if(currSlide > numImgs) {
+      currSlide = 1;
+    }
+    document.getElementById("slide-img").src = `images/img${currSlide}.jpg`;
+    document.getElementById("slide-number").innerHTML = `${currSlide},${numImgs}`;
+  }
+
+  prev.onclick = () => {
+    changeSlide(-1);
+  };
+
+  next.onclick = () => {
+    changeSlide(1);
+  };
+});
+
+function createSlide(currSlide, numImgs) {
+  let slide = document.getElementById("slide");
+
+  let prev = document.createElement("a");
+  prev.setAttribute("id", "prev");
+  prev.innerHTML = "❮";
+  slide.appendChild(prev);
+
+  let next = document.createElement("a");
+  next.setAttribute("id", "next");
+  next.innerHTML = "❯";
+  slide.appendChild(next);
+
+  let img = document.createElement("img");
+  img.setAttribute("src", `images/img${currSlide}.jpg`)
+  img.setAttribute("id", "slide-img");
+  slide.appendChild(img);
+
+  let slideNum = document.createElement("div");
+  slideNum.innerHTML = `${currSlide},${numImgs}`;
```

Comment:
just my opinion: what do you think of ${currSlide} of ${numImgs}

that is completely up to you

Reply:
done, also like it